tests/tmt/tests/e2e/test.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh -eux
+
+python3 ./make_test_results.py
+chmod -R 755 "$ARTIFACT_DIR" && cp -r "$ARTIFACT_DIR" .
```

Comment:
Why the chmod?

(You're marking all files within ARTIFACT_DIR ... as executable)

Reply:
Chmod is necessary as otherwise the artifacts fetched from the test vm (that are created by root) are inaccessible on the tmt runner machine that may not be running tests as root.
Although your concern about execute permissions on files themselves is valid, I will look into it. Looks like execute could be kept on ARTIFACT_DIR only, but not on the files.

Reply:
I think what you want is
```suggestion
chmod -R u=rwX,g=rX,o=rX "$ARTIFACT_DIR" && cp -r "$ARTIFACT_DIR" .
```
or maybe even
```suggestion
chmod -R a+r "$ARTIFACT_DIR" && cp -r "$ARTIFACT_DIR" .
```
if you are only concerned with read permissions for the files and folders.